src/main/resources/referenceImageTemplateWithScript.json

```diff
@@ -105,7 +105,7 @@
                     "osDisk": {
                         "name": "osdisk",
                         "vhd": {
-                            "uri": "[concat('http://',variables('storageAccountName'),'.blob.core.windows.net/',variables('storageAccountContainerName'),'/', variables('vmName'), copyIndex(), 'OSDisk.vhd')]"
+                            "uri": "[concat('http://',variables('storageAccountName'),'.blob.core.windows.net/',variables('storageAccountContainerName'),'/', variables('vmName'), uniquestring(resourceGroup().id), 'OSDisk.vhd')]"
```

Comment:
You should pass storageAccountName because the storage account name comes from the plugin now